test/main.js

```diff
@@ -157,13 +162,17 @@ describe('gulp-sass -- async compile', () => {
     const stream = sass();
 
     stream.on('error', (err) => {
-      // Error must include message body
```

Comment:
We use this style because error message formats are not part of the sass language spec and change over time. This style is more resilient to compiler changes as it simple asserts for key phrases.

Reply:
Yeah, but it's more troublesome to keep them in sync. For example, how should I test for the line with dart sass? Just check for number `2`?

Reply:
I'd opt for simple removing the line number. This is more the concern of the engine. All we really care about is that an error was produced.

Reply:
Please check now, it should be what you want. The line is still checked in another test, not sure if we should remove that too.